### PR TITLE
added ADR and updated C4 tech arch diagrams 

### PR DESCRIPTION
**Pull request by aje54**
I have added ADR 006 to record the decisions we made around RTT and Quals. Also changed the C4 Containers diagram to reflect the qualifications container (added Right To Teach and Quals Service as components - added assumptions around the tech as TBC)

### CODE REVIEW

File: docs/ADRs/adr-006-tech-choice-on-quals-right-to-teach.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1. Proposed tech for each service?
+2. One Front end application (in code repo terms) or two?
+3. How will we abstract away underlying data dependencies while we have high uncertainty on service design?
+4. Datasources?
+
+## Decision
+
+1. This is not a technical issue (i.e. either is fine from a tech POV), this is more a resource decision. TBC but the assumptions at the time of writing are:
+    Right To Teach (Ruby On Rails) + PostgreSQL
+    Qualifications (Ruby On Rails) + PostgreSQL
```
Comment (gunndabad) [inline docs/ADRs/adr-006-tech-choice-on-quals-right-to-teach.md:22]: Does each service needs it own Postgres DB if the API is doing the heavy lifting?

Comment (aje54) [reply docs/ADRs/adr-006-tech-choice-on-quals-right-to-teach.md:22]: maybe not, its just an assumption at the moment, the tech is marked as TBC

**Review by gunndabad — COMMENTED** [review]
Do we need to add ID into picture (as it now talks to DQT API directly)?

File: docs/ADRs/adr-006-tech-choice-on-quals-right-to-teach.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+2. Qualifications will proceed with a qualifications only repo, defer decision until RIght To Teach is closer to cutting code.
```
Comment (gpeng) [inline docs/ADRs/adr-006-tech-choice-on-quals-right-to-teach.md]:
```suggestion
2. Qualifications will proceed with a qualifications only repo, defer decision until Right To Teach is closer to cutting code.
```

**Review by gpeng — APPROVED** [review]

**Pull request merged**